app/models/actor.rb

```diff
@@ -0,0 +1,7 @@
+class Actor < ApplicationRecord
+  has_many :movie_actors
+  has_many :movies, through: :movie_actors
+  has_many :studios, through: :movie_actors
```

Comment:
While this is still "correct" linguistically, I think it is technically more accurate to say that an Actor `has_many :studios, through: :movies` -- and, I would only make this association when I absolutely need to. 
The reason I think it's more accurate to say it's through movies and not `movie_actors` is because you could potentially end up over-joining which would result in duplicated data in your result table. So, unless you need this `studios` association in your views etc., leave it alone and write a test for it when you need it.